neurom/core/dataformat.py

```diff
@@ -29,13 +29,13 @@
 """Data format definitions."""
 
 
-_COL_COUNT = 7
+_COL_COUNT = 4
 
 
 class COLS:
     """Column labels for internal data representation."""
     COL_COUNT = _COL_COUNT
-    (X, Y, Z, R, TYPE, ID, P) = range(_COL_COUNT)
+    (X, Y, Z, R) = range(_COL_COUNT)
```

Comment:
COLS.TYPE is used in `_remove_soma_initial_point`
COLS.P in  `_check_soma_topology_swc`